### ISSUE TITLE
Bug: Speed with prod mods dropped too far.

### ISSUE THREAD

**Issue by CricketBr** [DanielKote/Foreman2#33]
Eg In SeaBlock, turning spools of tinned wire into regular tinned wire in an electronics Assembler with 6 prod3.

The in-game sidebar says "Crafting Speed .80 (-80%=limit)".

Rate Calculator and Max Rate Calculator both say one machine will make 22.016 per second.

Foreman2 says it has speed of 10% and will only make 11.01 per second.

**Comment by PG-Fuury** [comment]
Related to this, Space Exploration's high-tier Productivity modules can give negative speeds leading to the program erroring from unexpected negative values.

**Comment by DanielKote** [comment]
I believe this has been fixed a while back - I have tested that exact scenario in the latest version and it does indeed produce 22 per second (not 11). Just in case I have added limits to all variables for the next version (productivity, pollution, speed, efficiency).
EDIT:
it was solved a while back but wasnt pushed to a release until now. In other words the issue has been solved in the dev13 release.
